src/Jumbotron.js

```diff
@@ -2,11 +2,21 @@ import React from 'react';
 import classNames from 'classnames';
 
 const Jumbotron = React.createClass({
+  propTypes: {
+    componentClass: React.PropTypes.node.isRequired
```

Comment:
A class isn't an Element in this case: https://jsfiddle.net/69z2wepo/10729/ so your example actually would fail 

also this example would not fail 
```
<Jumbotron componentClass={2}>
   <strong>foo</strong>
</Jumbotron>
```